line.js

```diff
@@ -148,19 +148,19 @@ H5P.Chart.LineChart = (function () {
 
       xAxis.tickSize([0]);
       xAxisG.attr('transform', 'translate(0,0)').call(xAxis);
-      //A lot of conditional moving here. If Y axis text is defined, we translate 40 px in the X direction. In the Y direction we translate downward the by current chartTitle height and line height
-      yAxisG
-          .attr('transform', 'translate(' + (isYAxisTextDefined ? 40 : 10) + ',' + (chartTextDefined ? chartTitleTextOffset : 0) + ')');
 
       yAxisG.call(yAxis
           .tickSize(-width, 0, 0)
           .ticks(getSmartTicks(d3.max(dataSet).value).count));
-
       //Gets all text element from the Y Axis
       var yAxisTicksText = yAxisG.selectAll('g.tick text')[0];
       //Gets width of last Y Axis tick text element
       var yAxisLastTickWidth = yAxisTicksText[yAxisTicksText.length-1].getBoundingClientRect().width;
 
+
+      //A lot of conditional moving here. If Y axis text is defined, we translate 40 px in the X direction. In the Y direction we translate downward the by current chartTitle height and line height
+      yAxisG
+          .attr('transform', 'translate(' + (isYAxisTextDefined ? 15 + yAxisLastTickWidth: 10 + yAxisLastTickWidth) + ',' + (chartTextDefined ? chartTitleTextOffset : 40) + ')');
```

Comment:
You are right, there are a lot of inline conditionals. Perhaps we can improve readability a bit by doing something like this:

```suggestion
      const xTranslation = (isYAxisTextDefined ? 15 + yAxisLastTickWidth : 10 + yAxisLastTickWidth);
      const xTranslation = (chartTextDefined ? chartTitleTextOffset : 40);

      yAxisG
          .attr('transform', `translate(${xTranslation}, ${yTranslation})`);
```

Also, what are 15, 10 and 40 the values of? Can we name them?